### PR TITLE
ath79: add support for KuWfi CPE830(D) / YunCore CPE830(D)

### PR DESCRIPTION
**Pull request by grosjo**
Board uses ath9k driver 

Short specification:
*  650/600/216 MHz (CPU/DDR/AHB)
* 2x 10/100 Mbps Ethernet, passive PoE support
* 64 MB of RAM (DDR2)
* 16 MB of FLASH
* 2T2R 2.4 GHz with external PA, up to 30 dBm (1000mW)
* 2x internal 14 dBi antennas
* 8x LED, 1x button
* No UART on PCB on some versions
* Display panel with 2x buttons (F/N) not supported (and not relevant in OpenWrt)- 

Flash instructions
* Connect PC with 192.168.0.141 to WAN port
* Install a TFTP server on your PC ('atftp' is doing the job for instance)
* Copy your firmware in the TFTP folder as upgrade.bin
* Power up device pushing the 'reset' button
* The device shall upload upgrade.bin, install it and reboot
* Device shall be booting on 192.168.1.1 as default

Signed-off-by: Joan Moreau <jom@grosjo.net>

### CODE REVIEW

File: target/linux/ath79/dts/qca9533_yuncore_cpe830.dts [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+                               reg = <0x040000 0x010000>;
+                       };
+
+                       partition@50000 {
+                               compatible = "denx,uimage";
+                               label = "firmware";
+                               reg = <0x050000 0xfa0000>;
+                       };
+
+                       art: partition@ff0000 {
```
Comment (neheb) [inline target/linux/ath79/dts/qca9533_yuncore_cpe830.dts:100]: art: is not needed.

Comment (grosjo) [reply target/linux/ath79/dts/qca9533_yuncore_cpe830.dts]: done

**Comment by Neustradamus** [comment]
Replaced by:
- https://github.com/openwrt/openwrt/pull/15969

**Comment by Neustradamus** [comment]
After several years, it has been merged in:
- https://github.com/openwrt/openwrt/pull/17279

**Pull request closed without merging**